server/embed/etcd.go

```diff
@@ -345,6 +347,18 @@ func print(lg *zap.Logger, ec Config, sc config.ServerConfig, memberInitialized
 		zap.String("auto-compaction-interval", sc.AutoCompactionRetention.String()),
 		zap.String("discovery-url", sc.DiscoveryURL),
```

Comment:
'endpoints` (to use the same word as clientv3) ? 

https://pkg.go.dev/go.etcd.io/etcd/client/v3 ?

Reply:
Thanks for the comment, but I'd prefer to keep the word for now, reasons:
1. Currently both V2 discovery and V3 discovery share the same parameter `--discovery`, we can change anything for V2 discovery for now;
2. Actually it's a little different from the endpoints used in clientv3. The discovery URL's format is something like `https://192.168.0.100:2379/57532354-625E-4A5F-AC73-D118A66CE270`, but the endpoints is an array of endpoint, and each of which is something like `https://192.168.0.100:2379`.

Reply:
Can I resolve this together with the other comment of reusing the same `client` config? @ptabor

Reply:
Yes. Seems right for another PR. 

It's unfortunate that it mixes connection address and business data (id of the cluster). 
Probably with V3 we should use to carry both the data-pieces:
```
  --discovery=57532354-625E-4A5F-AC73-D118A66CE270
  --discovery-endpoints="... , ..."
```

Reply:
Yes, I have exactly the same thought, so as to remove the suffix `UUID` from the discovery-url, and accordingly the `--discovery-endpoints` will be identical to the `--endpoints`.

Will kick off another session to track & resolve it soon.